packages/bundle-size/src/reporters/cliReporter.js

```diff
@@ -0,0 +1,76 @@
+const chalk = require('chalk');
+const Table = require('cli-table3');
+
+const { formatBytes } = require('../utils/helpers');
+
+/**
+ * @param {number} value
+ *
+ * @return {string}
+ */
+function getDirectionSymbol(value) {
+  if (value < 0) {
+    return '↓';
+  }
+
+  if (value > 0) {
+    return '↑';
+  }
+
+  return '';
+}
+
+/**
+ * @param {import('../utils/calculateDiffByMetric').DiffByMetric} diff
+ *
+ * @return {string}
+ */
+function formatDelta({ delta, percent }) {
+  if (delta === 0) {
+    return '';
+  }
+
+  const colorFn = delta > 0 ? chalk.red : chalk.green;
+
+  return colorFn(percent + getDirectionSymbol(delta));
+}
+
+/**
+ * @param {import('../utils/compareResultsInReports').ComparedReport} result
+ */
+module.exports = async function cliReporter(result) {
+  const table = new Table({
+    colAligns: ['left', 'right', 'right'],
+    head: ['Fixture', 'Before', 'After (minified/GZIP)'],
+
+    ...(process.env.NODE_ENV === 'test' && { style: { border: [], head: [] } }),
+  });
+
+  result.forEach(resultEntry => {
+    const { diff, gzippedSize, minifiedSize, name, packageName } = resultEntry;
+
+    if (diff.gzip.delta === 0 || diff.minified.delta === 0) {
```

Comment:
is it possible for delta values in minified but not in gzip ?

Reply:
Yes, I updated logic in it the shared function, 3687430.